modules/ROOT/pages/values-and-types/ordering-equality-comparison.adoc

```diff
@@ -144,3 +148,4 @@ If they have the same time and offset but different named time zones, they are s
 * Duration values cannot be directly compared.
```

Comment:
This page has specifics for other special types, so I think the vetor specifics can be here?